### PR TITLE
Adding time zone object related python calls.

### PR DESCRIPTION
**Pull request by Arunima19**

### CODE REVIEW

File: examples/ftd_api_scripts/timezone.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied.
+'''
+
+import sys
+from ftd_api_resources.timezone_utils import get_all_time_zones, post_time_zone, update_time_zone, get_time_zone, \
+    delete_time_zone
+from ftd_api_resources.access_token import get_access_token
+
+
```
Comment (stleary-cisco) [inline examples/ftd_api_scripts/timezone.py:20]: Please make these corrections:
time_zone_actions() should return True or False
Add params list to docstring, remove 'you must update...' line
main block should exit(0) or exit(1)

File: examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+express or implied.
+'''
+
+import sys
+from ftd_api_resources.timezoneobject_timezonesetting_association_utils import get_all_timezone_setting, \
+    update_timezone_setting, get_timezone_setting
+from ftd_api_resources.timezone_utils import post_time_zone
+from ftd_api_resources.access_token import get_access_token
+
+
```
Comment (stleary-cisco) [inline examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py:21]: Please make these corrections:
timezoneobject_timezonesetting_associated_actions() should return True or False
Add params list to docstring, remove 'you must update...' line
main block should exit(0) or exit(1)

File: examples/ftd_api_scripts/timezone.py (hunk 1 above)
Comment (Arunima19) [reply examples/ftd_api_scripts/timezone.py:20]: Done in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/8

File: examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py (hunk 2 above)
Comment (Arunima19) [reply examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py:21]: Done in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/8

**Pull request closed without merging**